package.json

```diff
@@ -15,24 +15,22 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "lerna run build",
     "changelog": "auto-changelog --remote upstream --commit-limit false",
-    "cnp": "NODE_ENV=test packages/create-project/bin/create-neutrino-project.js $(yarn random)",
     "link:all": "lerna exec yarn link",
     "lint": "node packages/neutrino/bin/neutrino lint",
     "precommit": "lint-staged",
     "random": "node -e \"process.stdout.write(require('crypto').randomBytes(8).toString('hex'))\"",
-    "release": "yarn build && lerna publish --force-publish=*",
-    "release:preview": "yarn build && lerna publish --force-publish=* --skip-git --skip-npm",
-    "test": "yarn test:all --no-verbose --match '!*@neutrinojs*'",
-    "test:all": "nyc --reporter lcov ava --verbose --fail-fast packages/*/test",
+    "release": "lerna publish --force-publish=*",
+    "release:preview": "lerna publish --force-publish=* --skip-git --skip-npm",
+    "test": "ava --fail-fast packages/*/test \"!packages/create-project/test\"",
```

Comment:
Could you document `yarn test` (and possibly `yarn test:create-project`, depending on how well it works locally) in `development.md`? :-)